test/integration/crud/estimated_document_count.test.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai';
+
+describe('Collection', () => {
```

Comment:
If we're going to add an integration test for this, I think we should add it with the others.  Adding tests for all falsy values should be as easy as adding `['estimatedDocumentCount', { } ]` to the array of commands here: https://github.com/mongodb/node-mongodb-native/blob/fe7c10249e5f72998c2417b244fb43ca6aed4f92/test/integration/node-specific/comment_with_falsy_values.test.ts#L27

Reply:
I've moved over to that test.